admin/sql/create_tables.sql

```diff
@@ -67,7 +67,17 @@ CREATE TABLE revision (
     id          SERIAL      NOT NULL,
     review_id   UUID,
     "timestamp" TIMESTAMP   NOT NULL,
-    text        VARCHAR     NOT NULL
+    text        VARCHAR,     
+    rating      SMALLINT CHECK (rating >= 0 AND rating <= 100)
```

Comment:
Do we even have ratings in MB that are not either 20, 40, 60, 80, or 100? My main concern is that scales are different and I don't know how we'll support other values set through the WS after they are imported into CB. Having https://tickets.metabrainz.org/browse/CB-248 implemented might help with this.

I'm fine with having the same storage as MBS does, but it doesn't seem right to me.